pkg/oc/cli/cmd/idle.go

```diff
@@ -117,14 +123,32 @@ func (o *IdleOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, args []st
 		return fmt.Errorf("resource names, selectors, and the all flag may not be be specified if a filename is specified")
 	}
 
+	o.ClientConfig, err = f.ToRESTConfig()
+	if err != nil {
+		return err
+	}
+
+	o.ClientSet, err = f.ClientSet()
+	if err != nil {
+		return err
+	}
+
+	o.Mapper, err = f.ToRESTMapper()
+	if err != nil {
+		return err
+	}
+
+	o.Typer = legacyscheme.Scheme
```

Comment:
This is likely to fail on non-groupified origin resources.  Use the read scheme we made.